doc/source/whatsnew/v0.23.0.txt

```diff
@@ -620,6 +620,7 @@ Other API Changes
 - Set operations (union, difference...) on :class:`IntervalIndex` with incompatible index types will now raise a ``TypeError`` rather than a ``ValueError`` (:issue:`19329`)
 - :class:`DateOffset` objects render more simply, e.g. ``<DateOffset: days=1>`` instead of ``<DateOffset: kwds={'days': 1}>`` (:issue:`19403`)
 - ``Categorical.fillna`` now validates its ``value`` and ``method`` keyword arguments. It now raises when both or none are specified, matching the behavior of :meth:`Series.fillna` (:issue:`19682`)
+- Making `to_datetime('today')` and `Timestamp('today')` consistent with one another (:issue:`19935`)
```

Comment:
say ``pd.to_datetime('today')`` is now consistent with  ``pd.Timestamp('today')``, this will return the local datetime.

Reply:
also add that pd.to_datetime previously returned a ``.normalized()`` datetime value.

Reply:
ok, added that